common/turbo_exec/turbo_start.tpt2

```diff
@@ -1,24 +1,24 @@
 
 ; This script registers a script for turbo exec, and exits as soon it confirms
```

Comment:
I would add more to this comment; this is (half of) the main API surface for TE2.0 so it should have a bit more to say.

Of particular importance to highlight is that you should call this with `executesync` and not `execute`, even though you might get away with the latter some of the time.

Things that would be best to add (unless you plan to put it in the README and not here): Some simple usage examples showing some basic cases for a script that only needs to do one thing (and doesn't care if it takes multiple frames), and also a script that wants frame-accurate timing in a loop.